apps/remix-ide-e2e/src/tests/solidityUnittests.test.ts

```diff
@@ -90,7 +90,7 @@ module.exports = {
     .scrollAndClick('*[data-id="testTabRunTestsTabRunAction"]')
     .pause(5000)
     .click('*[data-id="testTabRunTestsTabStopAction"]')
-    .pause(1000)
+    // .pause(1000)
```

Comment:
should we re add it?

Reply:
tests were failing with it, locally and on CI too